src/org/opendatakit/briefcase/model/BriefcaseAnalytics.java

```diff
@@ -0,0 +1,179 @@
+package org.opendatakit.briefcase.model;
+
+import com.brsanthu.googleanalytics.EventHit;
+import com.brsanthu.googleanalytics.GoogleAnalytics;
+import com.brsanthu.googleanalytics.GoogleAnalyticsConfig;
+import org.opendatakit.briefcase.buildconfig.BuildConfig;
+
+/**
+ *  This class is used to send usage information to ODK's Google Analytics account.
+ *
+ * <P>A library built by 'brsanthu' is utilized to streamline the
+ * process of contacting Google's Measurement Protocol API.</P>
+ * <P>This class also implements the Singleton pattern.</P>
+ *
+ * @author Joao Paulo Knox
+ * @see <a href="https://github.com/brsanthu/google-analytics-java">google-analytics-java</a>
+ */
+public class BriefcaseAnalytics {
+
+  private static final String GOOGLE_TRACKING_ID = BriefcasePreferences.GOOGLE_TRACKING_ID;
+  private static final String UNIQUE_USER_ID = BriefcasePreferences.getUniqueUserID();
+  private static final String EVENT_CATEGORY = "ODK Briefcase";
+  private static final String APPLICATION_NAME = BuildConfig.NAME;
+  private static final String APPLICATION_VERSION = BuildConfig.VERSION;
+
+  private static BriefcaseAnalytics instance;
+  private static final GoogleAnalyticsConfig CONFIG = new GoogleAnalyticsConfig();
```

Comment:
I got confused by the difference between constants and final references.
`static final int YEAR = 2017` would be a constant because of how (I think) it's a primitive type.
'static final MyClass myClass' would just be a final reference because of how it's a complex type which cannot be reassigned to some other instance/object.